reviews/graduation-coredns.md

```diff
@@ -0,0 +1,51 @@
+
+# CoreDNS Graduation Application
+
+CoreDNS joined CNCF in March 2017 as inception project and graduated as incubation project on February 2018
+
+It has grown significantly over time either as a DNS Server available in Cloud Native environment, either as a key component of Kubernetes project:
```

Comment:
It has grown significantly over time in maturity and user base, both as a DNS Server available in Cloud Native environment and as a key component of Kubernetes project:

Reply:
ok